src/transformers/models/owlvit/convert_owlvit_original_flax_to_hf.py

```diff
@@ -0,0 +1,376 @@
+import argparse
+import collections
+
+import torch
+import torch.nn as nn
+
+import jax
+import jax.numpy as jnp
+from clip_model import CLIP
```

Comment:
Where does this come from? Is this from the original repository?

Reply:
Yes, good points! I switched to using the openai clip instead